packages/i18nify-js/src/modules/currency/__tests__/formatNumber.test.ts

```diff
@@ -1,19 +1,19 @@
 import { getLocale } from '../../.internal/utils';
 import { setState } from '../../core';
-import { formatNumber } from '../index';
+import { CurrencyCodeType, formatNumber } from '../index';
 
 const nbsp = String.fromCharCode(160);
 const nnsp = String.fromCharCode(8239);
 
 describe('formatNumber', () => {
   it('should format the amount with default options', () => {
-    const result = formatNumber('1000.5', { currency: 'USD' });
+    const result = formatNumber('1000.5', { currency: 'USD' as CurrencyCodeType });
     expect(result).toBe('$1,000.50');
   });
 
   it('should format the amount with custom locale and currency display', () => {
     const result = formatNumber('1500', {
-      currency: 'EUR',
+      currency: 'EUR' as CurrencyCodeType,
```

Comment:
Why is typecasting required here now ?
How were these working before ?

Reply:
Not required. Added this coz there was an issue with local lint extension issue. Removing this.